lib/mcb/commands/providers/accredited_courses.rb

```diff
@@ -0,0 +1,30 @@
+name "accredited_courses"
+summary "Show courses this provider is the accredited body for"
+param :code, transform: ->(code) { code.upcase }
+
+run do |opts, args, _cmd|
```

Comment:
How about we replace this with the `accredited_course_sites` command above?

Reply:
done